backend/src/main/java/de/propertyfinder/backend/api/octoparse/OctoparseApiService.java

```diff
@@ -30,28 +29,51 @@
 
     private final AccessTokenUtil accessTokenUtil;
     private final ApiConnectionUtil apiConnectionUtil;
+    private final String BASEURL = "https://dataapi.octoparse.com/";
 
+    @Autowired
```

Comment:
Spring does not need `@Autowired` anymore, when there is only one constructor. And since the fields are final and you already use lombok, you can delete the constructor completely and annotate the class with `@RequiredArgsConstructor`.